rethinkdb/_export.py

```diff
@@ -185,6 +197,15 @@ def parse_options(argv, prog=None):
         if options.delimiter:
             parser.error("--delimiter option is only valid for CSV file formats")
 
+    if options.format == "jsongz":
+        if options.compression_level is None:
+            options.compression_level = -1
+        elif options.compression_level < 0 or options.compression_level > 9:
```

Comment:
My reasoning was: passing -1 is the same as not specifying the compression level, it's not really setting the `compression_level`.
Happy to make the change as suggested though, it might make it easier to switch between setting and not setting the compression level.

Reply:
I understand your reasoning, but then if someone passes `-1`, `if options.compression_level is None` is evaluated `False`, `if options.compression_level < 0 or options.compression_level > 9` is evaluated to `True` and raises an exception. Which is incorrect, since `-1` is an acceptable value.

If you have another suggestion on how to handle such situation, feel free to add it. Mine was just a potential suggestion on how to handle it.

All in all, It's no big deal to support `-1`, but could prevent some errors/exceptions, since I assume that the default value is an acceptable value.

Reply:
Updated as suggested.